tests/test_bench.py

```diff
@@ -606,49 +655,38 @@ def test_get_benchmark(
             assert gate_type in qiskit_helper.get_native_gates(gate_set_name) or gate_type == "barrier"
 
 
-def test_configure_end():
-    # delete all files in the test directory and the directory itself
+def test_create_benchmarks_from_config(output_path):
+    config = {
+        "timeout": 120,
+        "benchmarks": [
+            {
+                "name": "ghz",
+                "include": True,
+                "min_qubits": 2,
+                "max_qubits": 3,
+                "stepsize": 1,
+                "precheck_possible": True,
+            },
+        ],
+    }
+
+    Path("config.json").rename("old_config.json")
+
+    with Path("config.json").open("w") as f:
+        json.dump(config, f)
```

Comment:
Doesn't the `create_benchmarks_from_config` function take a path to the config?
Seems a bit over-complicated to me here.

Reply:
True, implemented that.